requirements.txt

```diff
@@ -7,3 +7,4 @@ prometheus_client
 psycopg2
 requests
 simplejson
+python-simplestreams
```

Comment:
We need to make sure these change are compatible with the execution environment where this will run. Basically, we need to ensure that the jenkins job which runs this is creating a virtualenv with --system-site-packages enabled.

Reply:
As far as I can see, jenkins job is ran using systemwide python in server team jenkins https://github.com/CanonicalLtd/server-jenkins-jobs/blob/master/metrics/metrics.yaml#L30.

Since distro-info is a dist-pacakge, that virtualenv flag is not necessary.